index.css

```diff
@@ -1,466 +1,709 @@
+  background: #fff;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+}
+
+.skills-list {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 100%;
+}
+
+.checkbox {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+}
+
+.checkbox > i {
+  font-size: 0.5rem;
+  color: lightgray;
+  margin: 0 1em;
+}
+
+.tech-list {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin: 0.25em 0;
+}
+
+.project-tech {
+  background-color: rgba(127, 140, 255, 0.5);
+  padding: 0.25em 0.5em;
+  border-radius: 0.5em;
+  color: var(--bold-purple);
+  margin: 0.5em 0.35em 0.5em 0;
+}
+
+.outlined-button {
+  display: block;
+  font-size: 18px;
+  padding: 0.5em 1em;
+  border: 1px solid var(--bold-purple);
+  color: var(--bold-purple);
+  border-radius: 5px;
+  margin-top: 1em;
+}
+
+.btn {
+  margin-top: 30px;
+  background-color: #fff;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+.btn:hover {
+  background: #6070ff;
+  color: white;
+}
+
+.btn:disabled {
+  color: #5e6c84;
+  border-color: #5e6c84;
+}
+
+.btn:active {
+  background-color: #2230d2;
+  color: white;
+}
+
+.project-description {
+  margin-top: 13px;
+  width: 95%;
+  max-width: 640px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: var(--text-color);
+}
+
+.test4 {
+  text-align: justify;
+  margin-bottom: 20px;
+}
+
+.about {
+  margin-top: 90px;
+  border-radius: 0 4em 0 0;
+  background-size: cover;
+  width: 375px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 24px;
+  left: calc(50% - 375px / 2);
+  background-color: #fff;
+}
+
+.about-icons {
+  color: var(--icon-color);
+  font-size: 20px;
+  padding-top: 30px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 50%;
+  margin-bottom: 30px;
+}
+
+.fa-angle-down {
+  display: flex;
+  justify-content: space-between;
+  color: black;
+}
+
+.fa-angle-right {
+  display: flex;
+  justify-content: space-between;
+  color: black;
+}
+
+.about-heading-small {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.about-skills-container {
+  flex-direction: column;
+  padding: 2em 1em;
+  width: 100%;
+}
+
+.about-skills-container > hr {
+  height: 2px;
+  background: #dfe1e6;
+  border: none;
+}
+
+.about-skill-heading {
+  justify-content: space-between;
+  padding: 2em 1em 2em 0;
+}
+
+.about-skill-body {
+  align-items: center;
+  justify-content: flex-start;
+  background-color: #f7f7f9;
+  border-radius: 0.5em;
+  margin: 0.5em 0;
+  padding: 0.5em;
+}
+
+.about-skill-text {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  padding-left: 1em;
+  line-height: 20px;
+  color: #253858;
+}
+
+.cont {
+  background-color: white;
+}
+
+footer {
```

Comment:
The footer top left has a pixelated view, kindly fix that. You can add a border-radius instead of using a background image.
<img width="651" alt="Screenshot 2023-02-25 at 8 55 18 PM" src="https://user-images.githubusercontent.com/25887253/221373372-fb0f101b-e542-44c4-b013-94b4a7562126.png">